sources/node-typescript/src/adapters/utils.ts

```diff
@@ -96,6 +96,12 @@ export function prepareParameters(request: KalturaRequest<any> | KalturaMultiReq
   );
 }
 
+export function safeJsonParse(obj: string): object {
+  try {
+    return JSON.parse(obj)
+  } catch (e) {/* noop */}
```

Comment:
Shouldn't it return the original string in case of error?

Reply:
as a standalone function I think better to return undefine if can't parse (cause return type should be object)..
but I can return the string in L:132 below (if not JSON) 👍